### ISSUE TITLE
OISD nsfw

### ISSUE THREAD

**Issue by cruxoooo** [AdguardTeam/HostlistsRegistry#480]
### Prerequisites

- [X] The blocklist is created specifically for DNS-level blockers;
- [X] The blocklist is regularly updated (at least 10 updates per month);
- [X] The blocklist is known and popular;
- [X] The blocklist satisfies [the repo requirements](https://github.com/AdguardTeam/HostlistsRegistry#-what-blocklists-can-be-added-here);

### Blocklist name

oisd nsfw

### Blocklist category

Other

### Homepage

https://nsfw.oisd.nl/

### Explanation

Oisd small and big are available in the custom DNS but oisd nsfw is missing.

**Comment by ShadowVoyd** [comment]
Yes please. This is one of the lists i would love to be able to use through adguard's services like adguard-dns.io
